lib/slide_hub/cloud/engine/azure.rb

```diff
@@ -100,9 +100,14 @@ def self.get_file_list(container, prefix)
         end
 
         def self.save_file(container, key, destination)
-          bs = ::Azure::Blob::BlobService.new
-          _blob, content = bs.get_blob(container, key)
-          File.open(destination, 'wb') { |f| f.write(content) }
+          begin
+            bs = ::Azure::Blob::BlobService.new
+            _blob, content = bs.get_blob(container, key)
+            File.open(destination, 'wb') { |f| f.write(content) }
```

Comment:
__[Reek]__

[UncommunicativeVariableName] SlideHub::Cloud::Engine::Azure#self.save_file has the variable name 'f'


<!-- Comment by [SideCI](https://sideci.com) -->